2.6/alpine/Dockerfile

```diff
@@ -1,6 +1,9 @@
 FROM alpine:3.16
 
-RUN apk add --no-cache ca-certificates mailcap
+RUN apk add --no-cache \
+	ca-certificates \
+	mailcap \
+	libcap
```

Comment:
Nitpick, but it would be nicer to have the package lists sorted here and elsewhere in this PR

Reply:
Sure, I have no strong preference for ordering. I tend to leave them in the order I thought to add them, personally. But not for any particular reason.

Reply:
Alright, I've reordered the packages.